lib/rpc.rb

```diff
@@ -0,0 +1,123 @@
+=begin
+  Copyright 2020 Nimiq community.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+=end
+
+require "oj"
+require "net/http"
+
+# :nodoc: all
+module ClientRPC
+  class Connect
+    attr_accessor :options, :uri, :ssl
+
+    DEFAULTS = {
+      host: "localhost",
+      port: 18332,
+    # dev: true,
+    }.freeze
+
+    def initialize(host)
+      @options = DEFAULTS.dup.merge(host.dup)
+      @uri = @options[:uri] ? URI(@options[:uri]) : URI(uri_check())
+    end
+
+    def request(name, *args)
+      puts "\n> #{name}: #{args.join(",")}" if options[:dev]
```

Comment:
Do you think something like [FakeWeb](https://github.com/chrisk/fakeweb) could be used for the tests so that the client library doesn't have to check whether it's being called from the tests in a development environment or from production code?

Reply:
@rraallvv let's turn this into a feature request 🙂